pybliometrics/utils/create_config.py

```diff
@@ -5,8 +5,8 @@
 
 
 def create_config(config_dir: Optional[Union[str, Path]] = None,
-                  keys: Optional[list[str]] = None,
-                  insttoken: Optional[str] = None
+                  custom_keys: Optional[list[str]] = None,
+                  custom_tokens: Optional[list[tuple[str, str]]] = None
```

Comment:
`insttoken` should be a list, I agree. But not a list of tuples. That will be too complex.